### PR TITLE
add podspec

### PR DESCRIPTION
**Pull request by iDay**
1.  run "pod install" under Example/ios
2. open Example.xcworkspace
3. run Example app in debug mode by click the run button

### CODE REVIEW

File: ios/ReactNativeCharts/RNCharts-Bridging-Header.h [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#import "React/RCTBridge.h"
+#import "React/RCTViewManager.h"
+#import "React/RCTUIManager.h"
+#import "React/UIView+React.h"
+#import "React/RCTBridgeModule.h"
+#import "React/RCTEventDispatcher.h"
+#import "React/RCTEventEmitter.h"
```
Comment (wuxudong) [inline ios/ReactNativeCharts/RNCharts-Bridging-Header.h]: missing

    #import "React/RCTFont.h"

**Comment by wuxudong** [comment]
This is my process:

1.    create-react-native-app Example
2.   cd Example
3.    yarn run eject
4.    yarn add https://github.com/yptech/react-native-charts-wrapper.git
5.    cd ios
6.    vi Podfile,   the content is exactly same to https://github.com/yptech/react-native-charts-wrapper/blob/c797bb2970d71ab2d9793ea6cf4e44ab176f6816/Example/ios/Podfile

7.    pod install

8.  open Example.xcworkspace

9.  run Example app,  but I get following compile error:

/xxx/github/Example/node_modules/react-native/ReactCommon/yoga/yoga/Yoga-internal.h:9:10: 'algorithm' file not found

/xxx/github/Example/node_modules/react-native/React/Views/UIView+React.h:11:9: Could not build module 'yoga'


screen shot 2018-07-30 at 12 56 34 pm

**Comment by iDay** [comment]
just create a empty swift file in project add create a bridging header



and then will meet 

just change #import <RCTAnimation/RCTValueAnimatedNode.h> to #import "RCTValueAnimatedNode.h" in RCTNativeAnimatedNodesManager.h
this is a bug in react native

**Comment by iDay** [comment]
if there is some thing wrong with ExampleTests, just remove ExampleTests from project

File: ios/ReactNativeCharts/RNCharts-Bridging-Header.h (hunk 1 above)
Comment (iDay) [reply ios/ReactNativeCharts/RNCharts-Bridging-Header.h]: that is no need here

**Comment by wuxudong** [comment]
These manual operation is annoying.

Is it possible automate all these procedures? 

I found a [doc](https://sandstorm.de/de/blog/post/react-native-managing-native-dependencies-using-xcode-and-cocoapods.html) about this, but I failed.

**Comment by iDay** [comment]
sorry there is no way to do that

**Comment by hungdev** [comment]
please write the guide install with pod install in next bump version.

**Pull request merged**